Expanding on the documentation for callable classes (#4301)
dart-lang/site-www: src/_guides/language/language-tour.md

}
```

## Callable classes

To allow an instance of your Dart class to be called like a function,
implement the `call()` method.

> The `call()` method is a special method
> that allows any class that defines it to emulate a function,
> while supporting the same functionality as normal [functions](#functions)
> such as parameters and return types.

Reviewer: **issue:** Tighten the language. The `...is a special method that...` does not aid in reader understanding. I would also cut this into shorter sentences.

```suggestion
The `call()` method allows any class that defines it to emulate a function.
This method supports the same functionality as normal [functions](#functions)
such as parameters and return types.
```
